src/payments/services/payment.service.ts

```
@@ -395,23 +395,89 @@ export class PaymentService {
         });
       }
 
-      this.processPaymentTargets(result.paymentIntentId, result.userId, result.metadata.contextId, result.metadata.purpose).catch(
+      this.paymentIntentService
+        .findById(result.paymentIntentId)
+        .then((intent) => this.schedulePostSuccessCertificateAndUnlock(intent))
+        .catch((error) => {
+          this.logger.error(
+            `Failed to load payment intent after webhook for ${result.paymentIntentId}: ${error.message}`,
+            error.stack,
```

ESLint

[error] Delete `,`

(prettier/prettier)